### PR TITLE
update crds

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: hack/copy-crds.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cp ./vendor/github.com/open-cluster-management/api/cluster/v1/0000_00_clusters.open-cluster-management.io_managedclusters.crd.yaml ./deploy/hub/managedcluster.crd.yaml
```
Comment (qiujian16) [inline hack/copy-crds.sh]: if we add this, we need to add script to verify also

Comment (zhiweiyin318) [reply hack/copy-crds.sh]: done

**Pull request merged**